package.json

```diff
@@ -58,25 +58,25 @@
     "stylus": "^0.51.1"
   },
   "devDependencies": {
-    "babel-eslint": "^3.1.23",
+    "babel-eslint": "^4.1.3",
     "chalk": "^1.1.0",
     "glob-manipulate": "^1.1.1",
     "gulp": "^3.9.0",
     "gulp-babel": "^5.1.0",
-    "gulp-eslint": "^0.15.0",
-    "gulp-filter": "^2.0.2",
+    "gulp-eslint": "^1.0.0",
```

Comment:
There are breaking changes in ESLint, we need to update our `.eslintrc` together with this update.